src/graphql/account.rs

```diff
@@ -585,6 +584,73 @@ impl AccountMutation {
 
         Ok(new_theme)
     }
+
+    /// Updates the current user's own account information.
+    #[allow(clippy::too_many_arguments)]
+    async fn update_my_account(
+        &self,
+        ctx: &Context<'_>,
+        password: Option<UpdatePassword>,
+        name: Option<String>,
+        department: Option<String>,
+        language: Option<String>,
+        theme: Option<String>,
```

Comment:
As `update_account` used them, why don't you use `UpdateName`, `UpdateDepartment`, `UpdateLanguage`, and `UpdateTheme` instead of `String`?